en/docs/references/adaptive-authentication-js-api-reference.md

```diff
@@ -128,6 +137,55 @@ executeStep(1,{
 };
 ```
 
+<a name = "step-options"></a>
+**Authentication step options**
+
+`         stepOptions        ` is an optional property that can be defined in the `         executeStep         `.
+This will add additional authentication options like `         forceAuth        `,
+`         subjectIdentifier        ` &
+`         subjectAttributeStep        ` . These attributes work as follows,
+
+<table>
+<thead>
+<tr class="header">
+<th>Attribute</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>forceAuth</td>
+<td>force a particular authentication step from adaptive authentication script.<br />
+**note: <b>onSuccess</b> callback should be provided for this to work.</td>
+</tr>
+<tr class="even">
+<td>subjectIdentifier</td>
+<td>Set the current authentication step as the subject identifier step.</td>
+</tr>
+<tr class="odd">
+<td>subjectAttributeStep</td>
+<td>Set the current authentication step as the subject attribute step.</td>
```

Comment:
```suggestion
<td>Set the current authentication step as the subject attribute step.  This will override the already configured subject attribute step.</td>
```